prime-router/docs/proposals/0008-reporting-entity-front-end.md

```diff
@@ -0,0 +1,104 @@
+System but has no way to get that data to ReportStream today.
+
+## Background
+
+As technical assistance teams are scaling up and out across the nation, the ReportStream’s web portal would be a good
+place for the assistance teams to send smaller, less tech-savvy submitting entities that need to get their data to the
+Public Health Department in a quick-and-(semi)-painless manner. One way to achieve this is to utilize ReportStream’s
+built-in features such as:
+
+  1. Authentication
+  2. ReportStream API integration 
+
+to create a form that accepts comma separated value (.CSV) files that can be
+sent to the ReportSteam API for the submitting entity’s respective Public Health Department to consume. This paper
+outlines a proposal on how this CSV Upload Submission feature would work and would be implemented.
+
+## Goals
+
+Create a new page on the ReportStream front-end which houses a simple web portal form that accepts CSV files, formatted
+in a constrained set of data formats for which ReportStream has sender schemas configured.
+
+This will involve segmenting users in the authentication service to allow for certain actions to be performed when given
+permissions to. For instance, if a submitting entity is also a State Public Health Department, they will have
+permissions to both see Daily Data (which is currently what they can see now), plus, they can have the permission to
+also Upload CSVs in their desired format. However, a submitting entity such as a hospital or school does not need the
+Daily Data screen, and so their permissions will solely be the Upload CSV feature.
+
+## Proposal
+
+We propose that to create a form in the ReportStream web portal that accepts CSV files for submitting entities to pass
+their data to their respective State Health Department.
+
+## Initial Implementation Plan
+
+This will involve 3 steps:
+1. **Authentication/Authorization:** The authentication service will have to be set up to send roles or permissions that
+pertain to the user/organization, and the web portal client, as well as the API, will need to consume these permissions
+and behave accordingly.
+
+    1. The permission that comes through can be called something like `sender`.
+
+    2. If it is necessary, also create a `receiver`
+permission to prevent organizations that do not receive reports from getting any reports. (This may not be necessary, as
+they would simply get a blank screen when they enter ReportStream.)
+
+    3. We will need to add a data element to the user scope returned from the authentication service that informs what format
+the sender is using to send data to ReportStream, i.e: `experity-emr-covid-19`. 
+
+2. **Creating the upload form in the web
+portal:** Creating the form is the easy part! Segmenting the users and having the client’s logic behave correctly may be
+involved with how ReportStream behaves today, but it is not expected to be too difficult. 
+
+    1. **Navigation:**
+     
```

Comment:
When you speak about navigation, I'm trying to imagine the page structure.  Could you describe the page structure starting from reportstream.cdc.gov home page. My concern is for what is common and separate with the download site. I would like to see a top-level bifurcation of pages say //reportstream.cdc.gov/sender/... is for the web sender and the //reportstream.cdc.gov/receiver/... is for the web receiver.